int32 d0_1;
int32 d0_2;
int32 d0_3;
int32 d0_4;
int32 d0_5;
int32 d0_6;
int32 d1_1;
int32 d1_2;
int32 d1_3;
int32 d1_4;
int32 d1_5;
int32 d1_6;
int32 d2_1;
int32 d2_2;
int32 d2_3;
int32 d2_4;
int32 d2_5;
int32 d2_6;